### DISCUSSION TITLE
Add fields to Taxon table for better full name generation

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5920] [Feature Requests]
**Describe the solution you'd like**

New fields in the `taxon` table:
- `cultivarNameAuthor`
- `fullNameCode` (Nomenclatural code)

**Requested By**
Ian Engelbrecht, NSCF South Africa

**Additional context**
Ian is working on resolving https://github.com/specify/specify7/issues/4182

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8201
